cime/src/externals/mct/mpi-serial/configure

```diff
@@ -3462,6 +3466,9 @@ case $ac_fc_v_output in
   *mGLOB_options_string*)
     ac_fc_v_output=`echo $ac_fc_v_output | sed 's/"-mGLOB[^"]*"/ /g'` ;;
 
+  # With Intel ifx, remove -loopopt=0
+  *-loopopt=0*)
+    ac_fc_v_output=`echo $ac_fc_v_output | sed 's/-loopopt=0//g'` ;;
```

Comment:
@naromero77 was this necessary because configure can't tell the difference between the old and new intel compilers?